ajax/libs/lumx/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "lumx",
+  "filename": "lumx.min.js",
+  "description": "The first responsive front-end framework based on AngularJS & Google Material Design specifications",
+  "keywords": [
+    "AngularJS",
+    "Material Design",
+    "Framework"
+  ],
+  "homepage": "http://ui.lumapps.com",
+  "version": "0.3.96",
+  "author": "LumApps",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/lumapps/lumx.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/lumapps/lumx.git",
+    "basePath": "dist",
+    "files": [
+      "ilumx*.+(js|css)",
```

Comment:
Oh! `"ilumx"` is a typo, should be `"lumx"` ...sorry!
I have corrected it and use a simple representation now.